packages/block-library/src/paragraph/test/edit.native.js

```diff
@@ -251,7 +251,7 @@ describe( 'Paragraph block', () => {
 			'wordpress.org'
 		);
 		fireEvent.changeText(
-			screen.getByPlaceholderText( 'Add link text' ),
+			screen.getByPlaceholderText( 'Add link text', { hidden: true } ),
```

Comment:
Noting this relates to https://github.com/WordPress/gutenberg/pull/51303#discussion_r1232058061, which refactors this test to interact with the element prior to it becoming inaccessible. I think it is fine for this PR to leave the `hidden` addition for now, but we should prefer avoiding its addition long term.